great_expectations_cloud/agent/actions/data_assistants/utils.py

```diff
@@ -21,6 +22,11 @@
     from great_expectations.core import ExpectationSuite
 
 
+class DataAssistantType(str, enum.Enum):
```

Comment:
@cdkini can we use this in an annotation somewhere?

Reply:
like this: https://github.com/great-expectations/cloud/pull/27/commits/bda01e2137657569bf4ca3bcf1324a0de639ad15 ?